pandas/tests/indexes/interval/test_interval.py

```diff
@@ -739,6 +743,10 @@ def test_get_loc_interval(self):
         pytest.raises(KeyError, self.index.get_loc, Interval(2, 3))
         pytest.raises(KeyError, self.index.get_loc,
                       Interval(-1, 0, 'left'))
+        # GH19349
+        assert self.index.get_loc(Interval(0, 1), method='exact') == 0
+        with pytest.raises(KeyError):
+            self.index.get_loc(Interval(0, 0.5), method='exact')
```

Comment:
Might be beneficial to also test than an `Interval` that only differs by `closed` returns a `KeyError` as well, e.g. `Interval(0, 1, closed='neither')`.